content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -207,6 +207,59 @@ same format as described above to the aggregated apiserver and set up the log in
 to pick up audit logs. Different apiservers can have different audit configurations and different
 audit policies.
 
+## Dynamic Auditing
+
+{{< feature-state feature-state state="alpha" >}}
+
+In Kubeernetes version 1.13, you can configure dynamic audit webhook backends  AuditSink API objects. 
+
+To enable dynamic auditingm you must set the following apiserver flags:
+
+- `--audit-dynamic-configuration`: the primary switch. When the feature is at GA, the only required flag.
+- `--feature-gates=DynamicAuditing=true`: feature gate at alpha and beta.
+
+When enabled, an AuditSink object can be provisioned:
+
+```yaml
+apiVersion: auditregistration.k8s.io/v1alpha1
+kind: AuditSink
+metadata:
+  name: <name>
+spec:
+  policy:
+    level: <level>
+    stages:
+    - <stage>
+  webhook:
+    throttle:
+      qps: <10>
+      burst: <15>
+    clientConfig:
+      url: <backend url>
+      service:
+        name: <service name>
+        namespace: <service namespace>
+      caBundle: <ca bundle>
+```
+
+For the complete API definition, see [the godoc](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/types.go).
+
+Multiple objects will exist as independent solutions. Dynamic policy will have no effect on backends configured from runtime flags. If webhook truncate options are set as runtime flags they will be applied to the dynamic backend.
```

Comment:
This paragraph is a bit confusing:
- Does the first sentence help explain the second and third? (relationship is not quite clear)
- As written, the second and third sentences seem to contradict each other. Dynamic policy has no effect if you configure from runtime flags? But if you set runtime flags they are applied to dynamic policy? As I write these questions I'm also wondering whether I'm confusing "policy" and "backend" -- but I think the paragraph, or possibly the intro content to this section, could be rewritten to make the relationship clearer.